cypress/integration/core/image-overlay.spec.ts

```diff
@@ -0,0 +1,41 @@
+describe("Image Overlay", () => {
+  beforeEach(() => {
+    cy.visit(
+      "http://localhost:56106/iframe.html?id=image-overlay--image-overlay"
+    );
+    cy.injectAxe();
+    cy.wait(500);
```

Comment:
@benheldoorn Waarom had je deze wait nodig?

Reply:
Om de images in te laden. Die komen van verder, dus dat kan wat tijd kosten. Als ze van wat dichter bij komen, kan deze pauze wel wat lager vermoed ik.

Reply:
Ik heb de afbeeldingen in `storybook-assets` geplaatst